src/scripts/devhub.zig

```diff
@@ -20,20 +21,42 @@ pub fn main(shell: *Shell, gpa: std.mem.Allocator, cli_args: CliArgs) !void {
     _ = gpa;
 
     var timer = try std.time.Timer.start();
-    try shell.zig("build install -Drelease", .{});
+    try shell.zig("build -Drelease -Dconfig=production install", .{});
     const build_time_ms = timer.lap() / std.time.ns_per_ms;
 
     const executable_size_bytes = (try shell.cwd.statFile("tigerbeetle")).size;
 
-    const run = Run{
+    const benchmark_result = try shell.exec_stdout("./tigerbeetle benchmark", .{});
+    const p90_ms = try get_measurement(benchmark_result, "query latency p90", "ms");
```

Comment:
I think it should be "batch latency" instead, as "query latency" refers to the second part of the benchmark when we measure the `get_account_transfers` function. Or maybe both??

Reply:
> Or maybe both

Agreed! Useful to catch glaring regressions in our query code too.

Reply:
Indeed, not sure what I was thinking when I decided to use `query latency` rather than `batch latency`. My current hypothesis is that I was thinking nothing at all, fixed!